Sources/BinaryJSON/UnsafePointer.swift

```diff
@@ -366,7 +366,7 @@ private extension BSON {
                 
                 let scopeBuffer = UnsafeMutablePointer<UnsafePointer<UInt8>>(allocatingCapacity:1)
                 
-                defer { scopeBuffer.deinitialize(); /*scopeBuffer.deallocCapacity(1) */ }
+				defer { scopeBuffer.deinitialize(); /*scopeBuffer.deallocCapacity(1) */ }
```

Comment:
I think my comment here still applies, plus you messed up the whitespace again :stuck_out_tongue: